i18n/fr.po

```diff
@@ -54,6 +54,9 @@ msgstr "Programme"
 msgid "Access to Template Generation"
 msgstr "Accès à la génération de modèles"
 
+msgid "Access to Import Data"
+msgstr "Importer des données"
```

Comment:
Same, access missing. If unsure about the correct French, we just leave it empty and English will be used. Same for PT below.